src/main/scala/cromwell/engine/io/gcs/GoogleCloudStorage.scala

```diff
@@ -23,8 +23,26 @@ import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
 object GoogleCloudStorage {
-  def apply(appName: String, credential: Credential, jsonFactory: JsonFactory, httpTransport: HttpTransport): GoogleCloudStorage = {
-    GoogleCloudStorage(new Storage.Builder(httpTransport, jsonFactory, credential).setApplicationName(appName).build())
+
+  val RefreshTokenOptionKey = "refresh_token"
+
+  lazy val cromwellAuthenticated: Try[GoogleCloudStorage] = {
+    for {
+      cromwellCredentials <- Try(GoogleCredentialFactory.fromCromwellAuthScheme)
+      conf <- GoogleConfiguration.gcloudConf
+    } yield apply(conf.appName, cromwellCredentials)
+  }
+
+  def userAuthenticated(workflowOptions: WorkflowOptions): Option[GoogleCloudStorage] = {
+    for {
+      conf <- GoogleConfiguration.gcloudConf.toOption
```

Comment:
not sure of the interplay between this and `cromwellAuthenticated`, but if `gcloudConf` is a `Failure` we don't log the error anywhere along the path and then `toOption` it out of existence